action/envelope.go

```diff
@@ -28,6 +29,10 @@ type (
 		Destination() (string, bool)
 		Cost() (*big.Int, error)
 		IntrinsicGas() (uint64, error)
+		BlobGas() uint64
+		BlobGasFeeCap() *big.Int
+		BlobHashes() []common.Hash
+		BlobTxSidecar() *types.BlobTxSidecar
```

Comment:
should it be part of the action or part of the envelope, if the returned `Action` is not just a payload, but AbstractAction Type?

Reply:
it should be part of `envelope`, there will be a clean-up of the interface in future PR,
this PR is just to unblock other PR